CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 * Fixed issue with simultaneous recognition of tap gesture. ([#4283](https://github.com/mapbox/mapbox-navigation-ios/pull/4283))
 * Fixed label localization to properly handle Simplified and Traditional Chinese. ([#4283](https://github.com/mapbox/mapbox-navigation-ios/pull/4283))
 * Allow simultaneous recognition of map- and annotation- handling gesture recognizers. ([#4283](https://github.com/mapbox/mapbox-navigation-ios/pull/4283))
+* Added `CarPlayManagerDelegate.carPlayManager(_:willAddRouteLineLayer:identifier:for:)`, `CarPlayManagerDelegate.carPlayManager(_:willAddRouteCasingLineLayer:identifier:for:)`,
```

Comment:
Is `identifier` parameter really needed in all these methods? Looks like parameter with `LineLayer` object is enough as it already contains `id` property.

Reply:
Thanks for the suggestion, I'll remove the identifier parameter.